src/senaite/jsonapi/v1/routes/registry.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+from senaite.jsonapi import api
+from senaite.jsonapi import request as req
+from senaite.jsonapi.v1 import add_route
+
+
+@add_route("/senaite-config-registry", "senaite.jsonapi.v1.senaite-config-registry", methods=["GET"])
+@add_route("/registry/senaite-config-registry", "senaite.jsonapi.v1.registry.senaite-config-registry", methods=["GET"])
+def get(context, request, username=None):
+    """Senaite configuration registry records route
+    """
+    registry_records = api.get_registry_records_by_keyword("bika")
+    registry_records.update(api.get_registry_records_by_keyword("senaite"))
```

Comment:
use the passed in keyword parameter `key` instead of hard coding these values in there